src/components/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../styles/ZIndexes.scss';
+@import '../../styles/Colors.scss';
+@import '../../styles/Mixins.scss';
+
+.dropdown {
+  position: sticky;
+
+  &__opener {
+    position: relative;
+  }
+
+  &__button {
+    text-transform: none !important;
+    flex: 1 !important;
```

Comment:
we need important here?

Reply:
I just checked on flex: 1 important is not needed,  but on text-transform: none important is  needed bec because material ui buttons have by default text-transform: uppercase

Reply:
it's fine! :)